Classes/model/CoreData/Alert.swift

```diff
@@ -14,7 +14,11 @@ extension Alert {
     switch alertSeverity {
     case .info, .warning: return .warning
     case .alert: return .alert
-    }
+    } 
+  }
+  
+  public var excludedStops: [String] {
```

Comment:
Please add a comment explaining what this is. What does it mean if an alert excludes stops? How is this typically used?

Reply:
Done